client/upe-onboarding-wizard/upe-preview-methods-selector/payment-method-checkbox/index.js

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useMemo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import styled from '@emotion/styled';
+import React from 'react';
+import { CheckboxControl, VisuallyHidden } from '@wordpress/components';
+import { Icon, info } from '@wordpress/icons';
+import PaymentMethodIcon from 'wcstripe/settings/payment-method-icon';
+import Tooltip from 'wcstripe/components/tooltip';
+import paymentMethodsMap from 'wcstripe/payment-methods-map';
+import PaymentMethodFeesPill from 'wcstripe/components/payment-method-fees-pill';
+import PaymentMethodCapabilityStatusPill from 'wcstripe/components/payment-method-capability-status-pill';
+import './style.scss';
+
+const InfoIcon = styled( Icon )`
+	fill: #949494;
+`;
+
+const InfoWrapper = styled.div`
+	flex-shrink: 0;
+	width: 20px;
+	height: 20px;
```

Comment:
I needed to set the width and height here, otherwise the SVG wasn't correctly positioned within its container